### ISSUE TITLE
Accessibility: Editable fields in the new dashboard should be at least 16px and higher contrast

### ISSUE THREAD

**Issue by patrys** [mirumee/saleor#3834]
Most inputs have a font size of 14px and they are dark grey. Tables also use a very small font. This makes the text hard to read for people with 20/20 vision so I imagine it's a nightmare for anyone with a medical condition such as astigmatism.

**Comment by maarcingebala** [comment]
cc @pwgryglak

**Comment by pwgryglak** [comment]
yes we've used elements that were in previous version of the dashboard to create the new version. I agree that the fonts should be bigger and in the project with new layout etc. font changes to bigger ones were accounted for. If we can we should change the base font to 16 and I'll adjust the project accordingly.
